k8s/config/oss/manager/controller_manager_config.yaml

```diff
@@ -15,6 +15,7 @@ leaderElection:
   leaderElect: true
   resourceName: 33405cb8.symphony
 syncIntervalSeconds: 180
+uniqueDisplayNameForSolution: false
```

Comment:
This is not enough. Please pass the settings from helm chart values. Otherwise the KAN project cannot use the same helm chart.